lib/streamlit/elements/map.py

```diff
@@ -166,8 +166,12 @@ def map(
             https://wiki.openstreetmap.org/wiki/Zoom_levels.
 
         use_container_width: bool
-            If True, set the chart width to the column width. This takes
-            precedence over the width argument.
+            Whether to override the figure's native width with the width of
```

Comment:
nit: one consideration here might be to use `map` instead of figure and/or chart.

Reply:
I'll say map/chart instead of figure/chart since it is a "scatterplot chart on top of a map."